### PR TITLE
feature: support null safety fully

### LINKED ISSUE #78: Add null safety support

### PR DESCRIPTION
**Pull request by dw2kim**
1. add full support for the null safety
2. upgrade the version to `1.0.0`
3. use cocoapad `1.10.1` instead of `1.4.0`

closes #78

### CODE REVIEW

**Comment by Hwan-seok** [comment]
👍🏻👍🏻
 If you do not mind, could you tell me when is the milestone?

File: example/ios/.symlinks/plugins/flutter_native_image [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/daewonkim/PrivateRepo/flutter_native_image/
```
Comment (btastic) [inline example/ios/.symlinks/plugins/flutter_native_image:1]: I don't like having personal data in this repository

File: example/ios/Flutter/flutter_export_environment.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Comment (btastic) [inline example/ios/Flutter/flutter_export_environment.sh:2]: As the comment in this line states, do not check this into version control. Also contains personal data about your computer / folder structure.

**Review by btastic — CHANGES_REQUESTED** [review]
Thanks for your PR. Please review the comments I made.

File: example/ios/Flutter/flutter_export_environment.sh (hunk 2 above)
Comment (dw2kim) [reply example/ios/Flutter/flutter_export_environment.sh:2]: Sounds good to me!

**Comment by dw2kim** [comment]
> Thanks for your PR. Please review the comments I made.

@btastic Thank you for the review. The changes were made so you can take a look at it again.

**Comment by btastic** [comment]
Thank you so much!

**Pull request merged**